src/main/java/seedu/toluist/ui/MainWindow.java

```diff
@@ -126,10 +127,20 @@ private void configureUndoKeyCombination() {
         FxViewUtil.setKeyCombination(getRoot(), keyCombination, handler);
     }
 
+    private void configureHistoryNavigationKeyPresses() {
+        String[] keyNames = new String[] { "up", "down"};
+        Arrays.stream(keyNames).forEach(keyName -> {
+            KeyCode keycode = getKeyCode(keyName);
+            String navigateCommand = "navigatehistory " + keyName;
```

Comment:
magic strings, "up", "down", "navigatehistory" should all go to private static final SOMETHING_SOMETHING respectively..